CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Resources/de.lproj/Localizable.strings

```diff
@@ -36,7 +36,14 @@
 "ginicapture.camera.documentValidationError.tooManyPages" = "Das Dokument darf nicht mehr als 10 Seiten haben";
 "ginicapture.camera.documentValidationError.wrongFormat" = "Nur Dateien vom Format jpg, tiff, png, gif oder pdf können analysiert werden";
 "ginicapture.camera.fileImportTip" = "Es können jetzt auch ganz einfach Dateien hochgeladen werden.";
-"ginicapture.camera.fileImportButtonLabel" = "Importieren";
+"ginicapture.camera.fileImportButtonLabel" = "Dateien";
+"ginicapture.camera.flashButtonLabel.On.Voice.Over" = "Blitz an";
+"ginicapture.camera.flashButtonLabel.Off.Voice.Over" = "Blitz aus";
+"ginicapture.camera.flashButtonLabel.On" = "An";
+"ginicapture.camera.flashButtonLabel.Off" = "Aus";
+"ginicapture.camera.thumbnail" = "TODO: Photo review";
```

Comment:
[Clarification] Do we need this label ginicapture.camera.thumbnail?
I don't see it in the design
![Uploading Screenshot 2022-09-27 at 18.15.04.png…]()

Reply:
@zladzeyka  I think so,  it's accessibility value  for voice over. Otherwise we don't have anything